python/vineyard/cli.py

```diff
@@ -254,14 +340,36 @@ def put_object(client, args):
                              f'{args.value} to vineyard server:\n{exc}')) from exc
     elif args.file is not None:
         try:
-            value = pd.read_csv(args.file)
+            value = pd.read_csv(args.file, sep=args.sep, delimiter=args.delimiter, header=args.header)
             client.put(value)
             print(f'{value} was successfully put to vineyard server')
         except BaseException as exc:
             raise Exception(('The following error was encountered while putting ' +
                              f'{args.file} as pandas dataframe to vineyard server:\n{exc}')) from exc
 
 
+def head(client, args):
+    """Utility to print the first n lines of a vineyard object."""
+    try:
+        value = client.get(as_object_id(args.object_id))
+        if isinstance(value, pd.DataFrame):
+            print(value.head(args.limit))
```

Comment:
else:
  print('Only support head a dataframe yet')

Or better you could fetch the meta first and check the type.